hikari/api/event_factory.py

```diff
@@ -697,6 +747,8 @@ def deserialize_voice_state_update_event(
             The shard that emitted this event.
         payload : hikari.internal.data_binding.JSONObject
             The dict payload to parse.
+        old_state: typing.Optional[hikari.voices.VoiceState]
+            The VoiceState object or 'builtins.None'.
```

Comment:
`builtins.None` should be enclosed in "`" not "'". Also, specifying that it can be None here is redundant as it is already in the typehint. (This also applies to all the other ones)